src/bp/ui-shared/src/Textarea/index.tsx

```diff
@@ -0,0 +1,48 @@
+import cx from 'classnames'
+import React, { FC, useEffect, useRef, useState } from 'react'
+
+import style from './style.scss'
+import { TextareaProps } from './typings'
+
+const Textarea: FC<TextareaProps> = props => {
```

Comment:
is this a simplified version of a RTE like slate,draft, and those kind of thing ?

Reply:
not sure I'm following you 🤔